server/db/index.js

```diff
@@ -0,0 +1,17 @@
+const db = require('./conn');
+const { Sequelize } = db;
+
+//Models
+const Product = require('./Product');
+const LineItem = require('./LineItem');
+const Order = require('./Order');
+
+//Associations
+LineItem.belongsTo(Product);
+LineItem.belongsTo(Order);
+
+db.sync({ force:true })
```

Comment:
We probably don't need force: true here. Since we created a seed.js file, which we're going to run once to seed and we'll test off that. 

If you elect to use force, then you must run the seed script as part of npm test, so we dont need to npm seed each time.